package.json

```diff
@@ -40,7 +40,7 @@
     "swipe-js-iso": "~2.0.1",
     "test-shape-closeness": "0.0.1",
     "visible-element": "~1.0.1",
-    "zooniverse-geordi-client": "~1.3.7"
+    "zooniverse-geordi-client": "zooniverse/geordi-client#panoptes-client"
```

Comment:
I think since we already update geordi-client that we can update this now to 1.3.10 ? Maybe it should be ^1.3.10 ?